src/pages/openPRs.tsx

```diff
@@ -0,0 +1,42 @@
+import { FunctionComponent, useState, useEffect } from 'react';
+import axios from 'axios';
+import Layout from '../components/Layout';
+import Navbar from '../components/navbar';
+import PullRequest from '../components/pullRequests';
+
+const openPRs: FunctionComponent = () => {
+    axios.defaults.baseURL = 'https://staging-api.realdevsquad.com/';
+
+    const [state, updateState] = useState(null);
+    useEffect(() => {
+        axios.get('/pullrequests/open')
+            .then((response) => {
+                updateState(response.data.pullRequests);
+                console.log(state);
```

Comment:
Better to remove "console.log(state)". 🙂

Reply:
Okok..will remove all the commented code